libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Test/SFSDKTestRequestListener.m

```diff
@@ -116,9 +114,9 @@ - (NSString *)waitForCompletion {
 - (void)configureAccountServiceDelegate
 {
     if (_serviceType == SFAccountManagerServiceTypeIdentity) {
-        _authMgr.idCoordinator.delegate = self;
+        [SFAuthenticationManager sharedManager].idCoordinator.delegate = self;
```

Comment:
Should these not follow a pattern of saving and restoring the original value?  Seems like it could have an impact on downstream tests, potentially.